docs/core-workflow.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Workflow
+---
+
+# Workflow
+
+## Folders structure
+
+Molecule is made of different parts that are organized under two hierarchical elements: Epics and Features. This terminology comes from Scrum, an agile method. Terminologies can differ depending on scrum implementations, but you can learn from it on https://www.atlassian.com/agile/project-management/epics-stories-themes.
+
+Warning: do not be confused between Agile Epics and redux-observables Epics. We'll use the term "Epic" with the Agile meaning here.
+
+### Epic
+
+An Epic is a set of features. We can imagine different parts on an IDE (text edition, tools execution, project management, etc...) and an Epic is related to those big parts. In Molecule we have the lib/ folder which contains the javascript code, and the first folders we see are the epics.
+
+We have many of them:
+
+- EventSystemEpic: which handles the file events
+- ExecutionControlEpic: which handles tools execution
+- ProjectSystemEpic: which handles management of projects
+- GlobalSystem: which links them all together
+
+
+
+Each Epic offers then features.
+
+### Features
+
+A feature is a part of an Epic, it's the first unit that would come to your mind if you had to describe what adds Molecule to Atom. "It adds a console, a terminal, etc...".
+
+Features follow all the same folder structure, which is directly related to Flux and specifically Redux.
+
+They can contain:
+
+- Actions: which contains redux action creators
+- Reducers: which contains redux reducers
+- Selectors: which contains some utils to select data from the related state
+- Containers: which contains react components which take data from the state and send them as props
+- Presenters: which contains visual components
+- Types: which contains consts, flow types and immutable Records
+- Fake: which contains fake data for unit testing
+- Styles: which contains some variables related to the UI
+- Layouts: which contains some old visual components
+- AtomLinks: which contains the functions that mount react components (ReactDOM.render) on Atom elements.
+- Model: which contains core functions for data management
+- Process: which contains files being launched as child processes
+
+## Code & Debug
+
+Let's say you want to add a new feature and that this feature contains a User Interface.
+
+### Storybook
```

Comment:
We no longer use Storybook in Molecule as our integration was outdated and broken.

Reply:
We have to make it work again, it's a really useful tool